docs/001_develop/03_client-capabilities/008_interaction/number-field.mdx

```diff
@@ -0,0 +1,357 @@
+---
+title: 'Number Field'
+sidebar_label: 'Number Field'
+id: client-interaction-number-field
+keywords: [interaction, interactive, input, numeric, field, form, data]
+tags:
+- interaction
+- interactive
+- input
+- numeric
+- form
+- data
+sidebar_position: 8
+---
+
+import NumberFieldDemo from '../../../../examples/ui/client-capabilities/interaction/number-field.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Number Field
+
+Use cases:
+* Numeric entry
+
+## Example
+
+<NumberFieldDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-number-field></rapid-number-field>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-text-field :value=${sync((x) => x.fieldValue)}>
+        </rapid-text-field>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      @observable fieldValue = '';
+      fieldValueChanged() {
+        console.log(this.fieldValue);
+      }
+    }
+    ```
+
+  </TabItem>
+  <TabItem value="react">
+
+    Declaration
+    ```html
+    <rapid-number-field></rapid-number-field>
+    ```
+    Usage
+    ```typescript
+    export function MyComponent() {
+      const handleChange = (event) => {
+        console.log(event.target.value)
+      };
+      return (
+        <rapid-number-field onChange={handleChange}></rapid-number-field>
+      );
+    }
```

Comment:
Can we use `useState` here too to actually track and update the field value please